roles/KubevirtRepoInfo/tasks/main.yml

```diff
@@ -1,11 +1,8 @@
 ---
 # tasks file for KubevirtRepoInfo
-- name: Extract the operator POD info
-  set_fact:
-    operator_pod: "{{ lookup('k8s', api_version='v1', kind='Pod', label_selector='name=kubevirt-ssp-operator') | from_yaml }}"
 - name: Extract the image name
   set_fact:
-    operator_image_name: "{{ operator_pod['spec']['containers'][0]['image'] }}"
+    operator_image_name: "{{ lookup('env', 'IMAGE_NAME') | default('quay.io/kubevirt/kubevirt-ssp-operator-container:latest', true }}"
```

Comment:
Aren't you missing a closing parenthesis?

Reply:
I do!